### ISSUE TITLE
[Chromatic] Implement a11y testing tool

### ISSUE THREAD

**Issue by Diaan** [sl-design-system/components#1775]
### Provide a general summary of the issue here

Turn this on:
https://www.chromatic.com/docs/accessibility/

- [ ] Make sure it doesn't fail unnecessarily for disabled components (also for what is visible in storybook, so CFA's wil have confidence in our components) (perhaps enable testing for non-refactored components, but not mark them as failed; since refactoring will fix a lot of axes issues most likely)
- [ ] See if it will work within our current chromatic plan
- [ ] Change setup of themes test so that it runs not on push but nightly or manually 

### 👤 Your name

Diana

**Comment by Diaan** [comment]
https://medium.com/@burakkara010/creating-a-recurring-github-actions-workflow-with-cron-jobs-15ce9e41f417

**Comment by a11ymiko** [comment]
We investigated use of a11y tests along with a11y dashboard for a month and finding is that with all a11y tests that we currently have (Storybook's automatic tests, Demos repository automated tests, manual tests done with each PR review) are sufficient.

Chromatic tests are finding the same issues that Storybook a11y plugin does. Cost of each a11y tests is the same as single Chromatic snapshot which would double our usage. 

Right now it's investment not worth taking, but we'll continue to monitor Chromatic a11y features.
